help_ukraine.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="/styles/style.css" rel="stylesheet"/>
+  <link href="/styles/help_ukraine.css" rel="stylesheet"/>
+</head>
+<body>
+  <div class="header-div">
```

Comment:
Since this `header-div` class occurs once in each of the HTML documents, it is okay to make it an ID rather than a class. (similar follows for a few other classes that occur once per HTML document.)